src/util/UserphraseSet.h

```diff
@@ -50,8 +50,17 @@ class UserphraseSet {
     const Userphrase& operator[](size_t index) const;
 
     void swap(UserphraseSet& set) noexcept;
+    
+    // shadow relative functionality
```

Comment:
I don't see any reason why the methods `insert_shadow` and `erase_shadow` should exist.